backend/curl_tests/create_account.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+# Create accounts for each group member
+curl --header "Content-Type: application/json" --request POST --data '{"type":"create_account","name":"aastha","password":"a_pass"}' http://localhost:8000/endpoint.php
+curl --header "Content-Type: application/json" --request POST --data '{"type":"create_account","name":"louie","password":"l_pass"}' http://localhost:8000/endpoint.php
+curl --header "Content-Type: application/json" --request POST --data '{"type":"create_account","name":"maddie","password":"m_pass"}' http://localhost:8000/endpoint.php
+curl --header "Content-Type: application/json" --request POST --data '{"type":"create_account","name":"mark","password":"m_pass"}' http://localhost:8000/endpoint.php
+curl --header "Content-Type: application/json" --request POST --data '{"type":"create_account","name":"tino","password":"t_pass"}' http://localhost:8000/endpoint.php
```

Comment:
Would this need to be formatted differently, now that we're planning on separating the type from the user object?

Reply:
Yes. I will update this alongside the pull request which fixes #38